BrowserKit/Sources/WebEngine/WKWebview/Extension/WKUserScriptExtension.swift

```diff
@@ -0,0 +1,34 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import WebKit
+import Foundation
+
+extension WKUserScript {
+    class func createInDefaultContentWorld(
+        source: String,
+        injectionTime: WKUserScriptInjectionTime,
+        forMainFrameOnly: Bool
+    ) -> WKUserScript {
+        return WKUserScript(
+            source: source,
+            injectionTime: injectionTime,
+            forMainFrameOnly: forMainFrameOnly,
+            in: .defaultClient
+        )
+    }
```

Comment:
Minor nit: these extensions seem slightly superfluous at first glance. They wrap the underlying WKUserScript initializers, but it's not obvious why they're needed (they don't seem to save any space or provide any convenience that I can see?).

Reply:
Yeah you're right 🤔 I put them in as it was done like that in the old codebase, it seems it was done just to call it as `createInDefaultContentWorld` rather than `WKUserScript.init` - Happy to change that if it's preferreable!